glauth/CHANGELOG.md

```diff
@@ -0,0 +1,207 @@
+
+
+* Enhancement - Implement group queries: [#22](https://github.com/owncloud/ocis/glauth/issues/22)
+
+   Refactored the handler and implemented group queries.
+
+   https://github.com/owncloud/ocis/glauth/issues/22
+   https://github.com/owncloud/ocis/glauth/pull/23
+
+# Changelog for [0.4.0] (2020-03-18)
+
+The following sections list the changes in ocis-glauth 0.4.0.
+
+[0.4.0]: https://github.com/owncloud/ocis/glauth/compare/v0.2.0...v0.4.0
+
+## Summary
+
+* Enhancement - Configuration: [#11](https://github.com/owncloud/ocis/glauth/pull/11)
+* Enhancement - Improve default settings: [#12](https://github.com/owncloud/ocis/glauth/pull/12)
+* Enhancement - Generate temporary ldap certificates if LDAPS is enabled: [#12](https://github.com/owncloud/ocis/glauth/pull/12)
+* Enhancement - Provide additional tls-endpoint: [#12](https://github.com/owncloud/ocis/glauth/pull/12)
+
+## Details
+
+* Enhancement - Configuration: [#11](https://github.com/owncloud/ocis/glauth/pull/11)
+
+   Extensions should be responsible of configuring themselves. We use Viper for config loading
+   from default paths. Environment variables **WILL** take precedence over config files.
+
+   https://github.com/owncloud/ocis/glauth/pull/11
+
+
+* Enhancement - Improve default settings: [#12](https://github.com/owncloud/ocis/glauth/pull/12)
+
+   This helps achieve zero-config in single-binary.
+
+   https://github.com/owncloud/ocis/glauth/pull/12
+
+
+* Enhancement - Generate temporary ldap certificates if LDAPS is enabled: [#12](https://github.com/owncloud/ocis/glauth/pull/12)
+
+   This change helps to achieve zero-configuration in single-binary mode.
+
+   https://github.com/owncloud/ocis/glauth/pull/12
+
+
+* Enhancement - Provide additional tls-endpoint: [#12](https://github.com/owncloud/ocis/glauth/pull/12)
+
+   Ocis-glauth is now able to concurrently serve a encrypted and an unencrypted ldap-port.
+   Please note that only SSL (no StarTLS) is supported at the moment.
+
+   https://github.com/owncloud/ocis/glauth/pull/12
+
+# Changelog for [0.2.0] (2020-03-17)
+
+The following sections list the changes in ocis-glauth 0.2.0.
+
+[0.2.0]: https://github.com/owncloud/ocis/glauth/compare/v0.3.0...v0.2.0
+
+## Summary
+
+* Change - Default to config based user backend: [#6](https://github.com/owncloud/ocis/glauth/pull/6)
+
+## Details
+
+* Change - Default to config based user backend: [#6](https://github.com/owncloud/ocis/glauth/pull/6)
+
+   We changed the default configuration to use the config file backend instead of the ownCloud
+   backend.
+
+   The config backend currently only has two hard coded users: demo and admin. To switch back to the
+   ownCloud backend use `GLAUTH_BACKEND_DATASTORE=owncloud`
+
+   https://github.com/owncloud/ocis/glauth/pull/6
+
+# Changelog for [0.3.0] (2020-03-17)
+
+The following sections list the changes in ocis-glauth 0.3.0.
+
+[0.3.0]: https://github.com/owncloud/ocis/glauth/compare/v0.1.0...v0.3.0
```

Comment:
Codacy found an issue: [[final-definition] Move definitions to the end of the file (after the node at line `202`)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)